packages/core/src/components/text/_text.scss

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+/*
+Text
+
+A component used to represent textual content.
+TODO BP add the docs you want here, I don't have the vision for this
+
+Styleguide components.text
+*/
+
+/*
+JavaScript API
+
+The `Text` component is available in the __@blueprintjs/core__ package.
+Make sure to review the [general usage docs for JS components](#components.usage).
+
+`Text` accepts and renders arbitrary children. It is intended that these children render as text.
```

Comment:
add something about the `title` attribute containing all the text content, to make it clear that this is meant for _text_ (as if the name isn't enough).